### ISSUE TITLE
Keras lab - wrong assertion?

### ISSUE THREAD

**Issue by gilcarmel** [udacity/sdc-issue-reports#199]
In the keras lab, shouldn't the assertions regarding accuracy test the accuracy on the last training epoch? e.g.
'''
accuracy = history.history['val_acc']
last_epoch_accuracy = accuracy[len(accuracy-1)]
assert(last_epoch_accuracy > 0.9, "The validation accuracy is: %.3f" % last_epoch_accuracy)
'''

**Comment by gilcarmel** [comment]
Oops, same idea but this code actually runs:
```
accuracy = history.history['val_acc']
last_epoch_accuracy = accuracy[len(accuracy)-1]
assert(last_epoch_accuracy > 0.9), "The validation accuracy is: %.3f" % last_epoch_accuracy
```

**Comment by bdholt1** [comment]
I noticed this as well and agree - the assertion should be on the last validation accuracy.

**Comment by philbort** [comment]
this is simpler:
`last_epoch_accuracy = accuracy[-1]`

**Comment by domluna** [comment]
The latest notebook uses the last history value for the tests. Also it tests for a higher value now as well :-)
